### PR TITLE
Reading the MirageOS tutorials

### PR DESCRIPTION
**Pull request by voodoos**
I started to read (and learn from !) the tutorial and will add to that PR the small corrections / suggestions I have.

### CODE REVIEW

File: tmpl/wiki/tutorial-lwt.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ##Basics
 
-The full Lwt manual is available [elsewhere](http://ocsigen.org/lwt/manual/), but the minimal stuff needed to get started is here.
+The full Lwt manual is available [elsewhere](https://ocsigen.org/lwt/5.1.2/manual/manual), but the minimal stuff needed to get started is here.
```
Comment (talex5) [inline tmpl/wiki/tutorial-lwt.md]: Looks like https://ocsigen.org/lwt would work here (that's what the site's header bar uses).
We should probably use https everywhere else in this file too.

File: tmpl/wiki/tutorial-lwt.md [hunk 2]
```diff
@@ -81,7 +81,8 @@ amount of time, say 1 and 2 seconds and then one prints "Heads", the other
 
 To sleep for some number of nanoseconds use `OS.Time.sleep_ns`, and to print to
 the console use `C.log`. Note that `OS` is a Mirage-specific module; if you are
-using Lwt in another context, use `Lwt_unix.sleep` and `Lwt_io.write`.
+using Lwt in another context, use `Lwt_unix.sleep` and `Lwt_io.write`. (You will
+also need to manually start the main evnt loop with `Lwt_main.run`.)
```
Comment (talex5) [inline tmpl/wiki/tutorial-lwt.md]:
```suggestion
also need to manually start the main event loop with `Lwt_main.run`.)
```

File: tmpl/wiki/tutorial-lwt.md (hunk 1 above)
Comment (voodoos) [reply tmpl/wiki/tutorial-lwt.md]: https://ocsigen.org/lwt redirects to http://ocsigen.org/lwt...

But https://ocsigen.org/lwt/5.2.0/manual/manual does not :-)

They should probably revise their server configuration... (and enforce https)
Meanwhile is it better to have a generic link that may survive the future versions but redirects to a non-secrure version of their site ? or an `https` one tied to a specific version of `lwt` ?

Comment (voodoos) [reply tmpl/wiki/tutorial-lwt.md]: I switched to https links and will open an issue at ocsigen.

**Comment by hannesm** [comment]
thank you.

**Pull request merged**